Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: watch clean fresh
+
+clean:
```

Comment:
There's already a clean script in package.json, why do we have this one here and why is it different?

Reply:
It's not very different honestly, during linking/unlinking deleting `node_modules`, `.turbo` and `build` recursively is something I do a lot so I thought it's probably good to have it solved with a `make` task.

Basically it's the same but doesn't depend on `rimraf`. Should I remove it?

Reply:
I don't know 🙂 I've just installed `rimraf` globally and then it's not a problem to run `pnpm clean` whenever needed. So not sure if we need a Makefile that kinda does the same thing 🤔

Reply:
You're right :) Dropped it in favor of `npm` scripts.